docs/usage/types/set_types.md

```diff
@@ -1,46 +1,40 @@
 `set`
 : allows `list`, `tuple`, `set`, `frozenset`, `deque`, or generators and casts to a set;
-  see `typing.Set` below for sub-type constraints
+  when a generic parameter is provided, the appropriate validation is applied to all items of the set
+
+`typing.Set`
+: handled the same as `set` above
 
 `frozenset`
 : allows `list`, `tuple`, `set`, `frozenset`, `deque`, or generators and casts to a frozen set;
-  see `typing.FrozenSet` below for sub-type constraints
-
-`deque`
-: allows `list`, `tuple`, `set`, `frozenset`, `deque`, or generators and casts to a deque;
-  see `typing.Deque` below for sub-type constraints
-
-`typing.Set`
-: see [Typing Iterables](#typing-iterables) below for more detail on parsing and validation
+  when a generic parameter is provided, the appropriate validation is applied to all items of the set
 
 `typing.FrozenSet`
-: see [Typing Iterables](#typing-iterables) below for more detail on parsing and validation
-
-`typing.Deque`
-: see [Typing Iterables](#typing-iterables) below for more detail on parsing and validation
+: handled the same as `frozenset` above
 
-## Constrained Types
```

Comment:
I removed this as we have a whole section about this stuff in `standard_types.md` that includes the `conset` and `confrozenset`.